CONTRIBUTING.md

```diff
@@ -16,6 +16,18 @@ How to setup the development enviroment to compile ValheimPlus yourself.
 1. Define Enviroment Variable `VALHEIM_INSTALL` with path to Valheim Install Directory  
    - example: `setx VALHEIM_INSTALL "C:\Program Files\Steam\steamapps\common\Valheim" /M`
 
+## Debugging with dnSpy
+
+Thanks to mono and unity-mono being open source, we patched and compiled our own mono runtime and enabled actual live debugging of the game and the mod itself with dnSpy.
+
+1. Download [dnSpy-net-win64](https://github.com/dnSpy/dnSpy/releases) and extract the exe.
+2. Load all assemblies from \<Valheim>\unstripped_corlib into dnSpy (just drag&drop the folder onto it).
+3. Load all assembly_* from \<Valheim>\valheim_Data\Managed into dnSpy (*do not load the publicized ones, they will not be loaded into the process and therefore can not be debugged*).
+4. Load ValheimPlus.dll from \<Valheim>\BepInEx\plugins into dnSpy.
+5. Copy .\libraries\Debug\mono-2.0-bdwgc.dll from this repo to \<Valheim>\MonoBleedingEdge\EmbedRuntime and overwrite the existing file.
```

Comment:
You use relative files paths in all prior steps, but step 5 refers to a 'this' directory. I think updating this to a more verbose directory would be a solid idea for 2 reasons.
1. It removes any potential confusion about where 'this' is, and makes steps more beginner friendly.
2. In the event that the CONTRIBUTING.md file ever moves location within the project, the location referenced by 'this' would be wrong.